src/seismometer/plot/mpl/binary_classifier.py

```diff
@@ -61,10 +61,15 @@ def evaluation(
         prevalence = prevalence / stats.loc[0, ["TP", "FP", "TN", "FN"]].sum()
 
     roc_data = ci_data["roc"]
+    roc_thresholds = roc_data["Threshold"]
+    # Normalize roc_data like stats; ignore infs inserted at beginning
+    if not (roc_thresholds[2:] > 2).any():
```

Comment:
```suggestion
    if not (roc_thresholds[2:] > 1).any():
```

1.5 is also greater than 1, at the very least `>=2` to make sure we catch that edge case.

Reply:
Updated.  This method was due to _old_ sklearn behavior, and the slicing skips that insertion regardless of value.

Reply:
And then deleted...